Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Development Dockerfile for bioformats2raw
+# -----------------------------------------
+
+# To install the built distribution into other runtimes
+# pass a build argument, e.g.:
+#
+#   docker build --build-arg IMAGE=openjdk:9 ...
+#
+
+# Similarly, the BUILD_IMAGE argument can be overwritten
+# but this is generally not needed.
+ARG BUILD_IMAGE=gradle:5.2.1-jdk8
+
+#
+# Build phase: Use the gradle image for building.
+#
+FROM ${BUILD_IMAGE} as build
+USER root
+RUN apt-get update -qq && apt-get install -y -qq zeroc-ice-all-dev libblosc-dev
+RUN echo Temporary && apt-get install -y -qq git maven
+RUN mkdir /bioformats2raw && chown 1000:1000 /bioformats2raw
+
+# Build all
+USER 1000
+
+COPY --chown=1000:1000 . /bioformats2raw
+WORKDIR /bioformats2raw
+RUN echo Temporary \
+ && git clone -b 0.0.2-beta git://github.com/saalfeldlab/n5-zarr \
+ && cd n5-zarr \
+ && mvn install -DskipTests
+RUN gradle build
+RUN cd build/distributions && rm bioformats2raw*tar && unzip bioformats2raw*zip && rm -rf bioformats2raw*zip
```

Comment:
Don't think the distributions will be available if the target is just `gradle build`

Reply:
They are. This is working.

Reply:
:+1:

That's a new one, just confirmed locally as well. Definitely not my memory of how the distribution plugin works but there have also been a ton of Gradle revisions since I first started using it. Learn something new every day. I added the very nice task tree plugin and ran a report for anyone who is interested in the various task dependencies:

```
callan@behemoth:~/code/bioformats2raw$ ./gradlew build taskTree

> Task :taskTree

------------------------------------------------------------
Root project
------------------------------------------------------------

:build
+--- :assemble
|    +--- :distTar
|    |    +--- :jar
|    |    |    \--- :classes
|    |    |         +--- :compileJava
|    |    |         \--- :processResources
|    |    \--- :startScripts
|    +--- :distZip
|    |    +--- :jar
|    |    |    \--- :classes
|    |    |         +--- :compileJava
|    |    |         \--- :processResources
|    |    \--- :startScripts
|    \--- :jar
|         \--- :classes
|              +--- :compileJava
|              \--- :processResources
\--- :check
     +--- :checkstyleMain
     |    \--- :classes
     |         +--- :compileJava
     |         \--- :processResources
     +--- :checkstyleTest
     |    +--- :classes
     |    |    +--- :compileJava
     |    |    \--- :processResources
     |    \--- :testClasses
     |         +--- :compileTestJava
     |         |    \--- :classes
     |         |         +--- :compileJava
     |         |         \--- :processResources
     |         \--- :processTestResources
     \--- :test
          +--- :classes
          |    +--- :compileJava
          |    \--- :processResources
          \--- :testClasses
               +--- :compileTestJava
               |    \--- :classes
               |         +--- :compileJava
               |         \--- :processResources
               \--- :processTestResources


To see task dependency tree for a specific task, run gradlew <project-path>:<task> <project-path>:taskTree [--no-repeat] [--task-depth <depth>]
Executions of all tasks except for taskTree will be skipped. They will be used for building the task graph only.

Add --no-repeat to prevent printing a subtree of the same task more than once.

For example, try running gradlew :build :taskTree

BUILD SUCCESSFUL in 0s
1 actionable task: 1 executed
```

Reply:
Any reason to use an old version of gradle as the default?
I have tested with ``gradle:6.3.0-jdk8`` and it builds fine . 
Note that the following commands need to be added in order to work in that case
```
ENV TZ=Europe/London
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Reply:
I assume no reason other than it's the version we use in `gradlew` ; the `build.gradle` is definitely old style in this repository and would probably work even with gradle 2.x